src/less/chat.less

```diff
@@ -136,8 +143,19 @@
     color: @sfrpgRed;
     font-weight: bold;
   }
+  .message-metadata {
+    flex: 2;
+    text-align: right;
+    font-size: 12px;
+  }
+  .flavor-text,
+  .whisper-to {
```

Comment:
Same with this block. These are already defined in core `css` styling so aren't needed here.

Reply:
I think these were leftover from when I was trying to figure out why the `.whisper` and `.blind` text wasn't appearing on the GM's view.